test-cases/terraform/hcl_language_complexity/using_count_and_ternary_expr/main.tf

```diff
@@ -30,3 +30,38 @@ resource "aws_iam_user_login_profile" "jill" {
   user    = aws_iam_user.jill.name
   pgp_key = "keybase:some_person_that_exists"
 }
+
+resource "aws_security_group_rule" "trust-rules-dev" {
```

Comment:
Why did you add these? The goal of this test case is to see if the tool can capture the use of count and a ternary expression.

Reply:
Because tfsec doesn't support the issue you used in the example - the reason being that we don't consider that to be a valid check. It does however support counts, so needed an example block that would demonstrate that.